### ISSUE TITLE
[Bug]: Nginx Can't Start 'unknown "herd_sock_" variable'

### ISSUE THREAD

**Issue by RiloArbabillah** [beyondcode/herd-community#502]
### Operating system version

macOS Sonoma 14.3.1

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.5.0

### PHP Version

PHP 8.3.3 (cli) (built: Feb 16 2024 00:15:01) (NTS)

### Bug description

Unable to start nginx, already tried reinstalling herd.

### Steps to reproduce

start herd

### Relevant log output

```shell
2024/03/25 23:20:08 [emerg] 93373#1112802: unknown "herd_sock_" variable
```

**Comment by mpociot** [comment]
Which PHP versions do you have installed?

The issue is coming from a site-specific configuration file. 
You should see more information in your nginx-error.log located here: `~/Library/Application Support/Herd/config/nginx`

Did you maybe migrate from Valet using an isolated site that used a PHP version that is unavailable in Herd (<7.4)?

**Comment by RiloArbabillah** [comment]
yup. i found the problem, some of my site is missing the php version after isolation.
thank you for the support.

<img width="330" alt="Screenshot 2024-03-27 at 13 52 19" src="https://github.com/beyondcode/herd-community/assets/45306630/49ee6aee-ae32-44f5-bc7b-c774e2a3a811">
